### PR TITLE
circle: install minimal python

### PR DESCRIPTION
**Pull request by iMichka**

### CODE REVIEW

**Comment by sjackman** [comment]
Can you make the the modifications to `libx11` and `libxcb` in this same PR?

**Comment by maxim-belkin** [comment]
python2.7-minimal

File: .circleci/config.yml [hunk 1]
```diff
@@ -10,6 +10,11 @@ jobs:
       HOMEBREW_VERBOSE: 1
       HOMEBREW_VERBOSE_USING_DOTS: 1
     steps:
+      - run:
+          name: Install minimal python for libxcb build
```
Comment (maxim-belkin) [inline .circleci/config.yml:14]: Thank you for this line

**Comment by iMichka** [comment]
```
==> brew audit linuxbrew/xorg/libxcb --online
Error: 2 problems in 1 formula
linuxbrew/xorg/libxcb:
  * The URL https://www.x.org/ is not reachable
  * Stable: The URL https://xcb.freedesktop.org/dist/libxcb-1.12.tar.bz2 is not reachable
==> FAILED
```

Just audit failures which we can ignore.

File: .circleci/config.yml [hunk 2]
```diff
@@ -10,6 +10,11 @@ jobs:
...
+          command: |
+            sudo apt-get update
+            sudo apt-get install --yes --force-yes python2.7-minimal
```
Comment (maxim-belkin) [inline .circleci/config.yml:17]: Isn't `-y` sufficient (instead of `--yes --force-yes`)?

Comment (iMichka) [reply .circleci/config.yml:17]: I think `-y` == `--yes`, that's all?

**Comment by iMichka** [comment]
I there are further changes/comments, I can make a second PR. I merged this just to see if it helps build a python bottle.

**Comment by sjackman** [comment]
Two more changes are needed. `libx11` and `libxcb` need to be modified:
```ruby
depends_on "python@2" => :build unless which "python2.7"
```
Linuxbrew/core circle.yml needs to `apt-get install python2.7-minimal`.

**Comment by iMichka** [comment]
> Linuxbrew/core circle.yml needs to apt-get install python2.7-minimal.

No, it seems to be working fine without.

I'll open a PR to clean up the syntax.

**Comment by sjackman** [comment]
Wonderful! Though I'm not quite sure why `python2.7` is already in the path.

**Pull request closed without merging**